### PR TITLE
Rename Tenant Networks to Storage Endpoints

### PR DESCRIPTION
**Pull request by jankodera-ps**
##### SUMMARY
This commit propagates changes in underlying Python SDK,
where Tenant Networks were renamed to Storage Endpoints.

Fixes internal issue HM-4757.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
fusion_tn.py -> fusion_se.py (+ code changes in said file)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,13 @@ Purestorage.Fusion Release Notes
 
 .. contents:: Topics
 
+unreleased
```
Comment (sdodsley) [inline CHANGELOG.rst:7]: Do not change this file. It is automatically updated when the Collection is created and released

**Review by sdodsley — CHANGES_REQUESTED** [review]
This PR is a problem. 
Because the collection has already been released you can't just rename an existing module. This may break playbooks already in use.
There will be rules in Ansible about how to do what you want to achieve, 
Check out the main Ansible docsite

**Comment by sdodsley** [comment]
Also, this is NOT a bugfix.
As you are adding a new module it is a major change that will be picked up by the release process.
I will not allow this PR to be merged until you understand and have the read the document on Ansible Collections that was created.

**Comment by sdodsley** [comment]
As far as Ansible semantics are concerned this is not a bug fix. Renaming a module is not allowed anyway. You would have to create a new one, leave the old one there and do some deprecation and aliasing as per Ansible standards

You would also need a release note stating which modules you changed and why. This release note will be used by the automated system to create the CHANGELOG, so that is why you shouldn't change that.

Check out how we do it with other Collections, such as FlashBlade: https://github.com/Pure-Storage-Ansible/FlashBlade-Collection/pulls?q=is%3Apr+is%3Aclosed

**Comment by jankodera-ps** [comment]
Right, thanks for the explanation, closing this PR for now. Will follow up with another version.

**Pull request closed without merging**